Providers/Modules/Plugins/ChangeTracking/plugin/changetracking_lib.rb

```diff
@@ -5,9 +5,18 @@
 require 'logger'
 
 require_relative 'oms_common'
+require_relative 'oms_configuration'
 
 class ChangeTracking
 
+    @@proxy_conf_path = '/etc/opt/microsoft/omsagent/conf/proxy.conf'
```

Comment:
You never use this value; why is it here?

This is the old proxy path: you should use the OMS configuration method get_proxy_config with the new proxy path (/etc/opt/microsoft/omsagent/proxy.conf) to correctly retrieve proxy info if you need it.